<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorruptorClass</class>
 <widget class="QMainWindow" name="CorruptorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Corruptor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="corruptionTypeCombox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Substitution</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Insertion</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Deletion</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Swap</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="inFileSelectButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QLabel" name="corruptionTypeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Corruption Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="numCorruptionsLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string># Corruptions</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numCorruptionsSpinner">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>2147483647</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="inputFileLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File</string>
    </property>
   </widget>
   <widget class="QLabel" name="outFileLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="corruptButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CORRUPT</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="numProtectedBytesSpinner">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2147483647</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="numProtectedBytesLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Header Bytes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inFileLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="outFileSelectButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outFileLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>inFileLineEdit</tabstop>
  <tabstop>inFileSelectButton</tabstop>
  <tabstop>corruptionTypeCombox</tabstop>
  <tabstop>numCorruptionsSpinner</tabstop>
  <tabstop>numProtectedBytesSpinner</tabstop>
  <tabstop>outFileLineEdit</tabstop>
  <tabstop>outFileSelectButton</tabstop>
  <tabstop>corruptButton</tabstop>
 </tabstops>
 <resources>
  <include location="Corruptor.qrc"/>
 </resources>
 <connections/>
</ui>
